### PR TITLE
docs(sidenav): leftNav text isn't hiding when it should

### PR DESCRIPTION
**Pull request by Splaktar**

### CODE REVIEW

File: src/components/sidenav/demoBasicUsage/index.html [hunk 1]
```diff
@@ -4,11 +4,10 @@
   <section layout="row" flex>
 
     <md-sidenav
-        class="md-sidenav-left"
+        class="md-sidenav-left md-whiteframe-4dp"
```
Comment (DevVersion) [inline src/components/sidenav/demoBasicUsage/index.html]: Not sure, why you've changed the directive to the class.

`md-whiteframe="4"` should work fine and also shows the API in use.

Comment (Splaktar) [inline src/components/sidenav/demoBasicUsage/index.html]: That seems to be covered in the md-whiteframe docs. Do we need to cover both usages of md-whiteframe in the md-sidenav docs? It seems like it only adds to the confusion and complexity of learning about sidenavs.

Comment (Splaktar) [inline src/components/sidenav/demoBasicUsage/index.html]: @DevVersion updated only to fix text hiding. Both forms of `md-whiteframe` API usage (attribute and class) preserved.

Comment (DevVersion) [inline src/components/sidenav/demoBasicUsage/index.html]: Yeah, looks fine now :+1: thx.

**Pull request closed without merging**